examples/autokey_example/main.tf

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "autokey" {
+  source = "../.."
+
+  project_id            = var.project_id
+  keyring               = "example-keyring"
+  location              = var.location
+  keys                  = []
```

Comment:
this should go away if you make the submodule independent; along with several others that aren't needed for autokey

Reply:
I'll turn the submodule independent, as I mentioned in our other thread! Then we can cut this off!